workflows/prognostic_c48_run/runtime/steppers/prescriber.py

```diff
@@ -62,22 +66,26 @@ def __init__(
                 containing data arrays to be prescribed
             variables: a mapping from variable names returned by the
                 `time_lookup_function` to the standard names to be prescribed
+            tendency_variables: mapping from tendency variable names returned by the
+                `time_lookup_function` to the standard tendency names used to update
+                the state
         """
         self._communicator = communicator
         self._time_lookup_function = time_lookup_function
         self._variables = variables
+        self._tendency_variables = tendency_variables or {}
 
     def _open_prescribed_timestep(self, time: cftime.DatetimeJulian) -> xr.Dataset:
         ds = scatter_within_tile(time, self._time_lookup_function, self._communicator)
-        return ds.rename(self._variables)
+        return ds.rename(**self._variables, **self._tendency_variables)
```

Comment:
Previously your PR made the rank available to the stepper. Why was that and is it no longer necessary when using the prescriber?

Reply:
The original version was going to use a "predictor" that looked up the correction from a dataset and applied it within each rank. That needed rank info since all our current usages of ML are column local and don't care about x/y/tile location. After our discussion, I figured it was more straightforward to add a new postphysics stepper for bias correction so that it could use the existing functionality of the prescriber to assigning the local corrections.

Reply:
Got it, thanks for the clarification